index.html

```diff
@@ -243,6 +243,32 @@ <h1>
                 </div>
             </div>
         </section>
+        <section class="contact-me">
+
+            <div class="contact-text">
+                <h2>Contact me</h2>
+
+                <p>If you have an application you are interested in developing, a feature that you need built
+                    or a project that you need coding. I'd love to help with it.</p>
+            </div>
+
+            <div class="form">
+                <form action="https://formspree.io/f/xoqrdelo" method="post">
+                    <ul>
+                        <li><input type="text" name="name" placeholder="Your Name" maxlength="30" required></li>
+                        <li><input type="email" name="email" placeholder="Your Email" required></li>
+                        <li><textarea name="message" placeholder="Write your message here" maxlength="500"
```

Comment:
- [ ]  According to the Figma design, The placeholder texts should be aligned in the left in input fields, Right now they are aligned in the center. Please align them left in the input fields and make sure it matches the figma design.